test/Error/rlexe.xml

```diff
@@ -30,6 +30,8 @@
     <default>
       <files>stack.js</files>
       <baseline>stack.baseline</baseline>
+      <!-- xplat-todo: Understand why this test is failing on OS X -->
+      <tags>exclude_mac</tags>
```

Comment:
Alternatively, I could merge my change in, you could rebase on top of this and use it to verify that the Jenkins PR is working- what do you prefer?